libgit/repo.go

```diff
@@ -203,6 +205,52 @@ func UpdateRepoMD(ctx context.Context, config libkbfs.Config,
 	return nil
 }
 
+func normalizeRepoName(repoName string) string {
+	return strings.TrimSuffix(strings.ToLower(repoName), gitSuffixToIgnore)
+}
+
+func takeConfigLock(
+	fs *libfs.FS, tlfHandle *libkbfs.TlfHandle, repoName string) (
+	f billy.File, err error) {
```

Comment:
I wonder whether you should just return an `io.Closer` here, to make it clear how this is to be used.